libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/trip/notification/NotificationTurnIconResources.kt

```diff
@@ -1,8 +1,8 @@
-package com.mapbox.navigation.trip.notification.internal
+package com.mapbox.navigation.base.internal.trip.notification
 
 import androidx.annotation.DrawableRes
+import com.mapbox.navigation.base.R
 import com.mapbox.navigation.base.maneuver.model.BaseTurnIconResources
-import com.mapbox.navigation.trip.notification.R
 
 class NotificationTurnIconResources private constructor(
```

Comment:
Maybe we should expose a resource api similar to [`MapboxLaneIconsApi`](https://github.com/mapbox/mapbox-navigation-android/blob/623705630c6b2cf20e16db82bbd56c5264bfdde4/libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/api/MapboxLaneIconsApi.kt#L15-L36)

We did this for android auto as well https://github.com/mapbox/mapbox-navigation-android/pull/4729

Reply:
`TurnIconHelper` looks like such an API. We can think of making it public.